### ISSUE TITLE
Run time error in mnist example

### ISSUE THREAD

**Issue by shersoni610** [pytorch/pytorch#30366]
Hello,

I downloaded the libtorch c++ for Ubuntu 18.04 running Cuda 10.1.

While running the mnist example: I get the following error:

csverma@blackhole:$ ./mnist 
CUDA available! Training on GPU.
Warning: torch::nn::FeatureDropout module is deprecated.Use Dropout{2,3}d instead. (FeatureDropoutImpl at ../../torch/csrc/api/src/nn/modules/dropout.cpp:65)
terminate called after throwing an instance of 'c10::Error'
  what():  max_pool2d_with_indices_out_cuda_frame failed with error code 0 (max_pool2d_with_indices_out_cuda_template at /pytorch/aten/src/ATen/native/cuda/DilatedMaxPool2d.cu:403)
frame #0: c10::Error::Error(c10::SourceLocation, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) + 0x6a (0x7f0cf4f7892a in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libc10.so)
frame #1: at::native::(anonymous namespace)::max_pool2d_with_indices_out_cuda_template(at::Tensor&, at::Tensor&, at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, bool) + 0x256e (0x7f0cf7f2151e in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #2: at::native::max_pool2d_with_indices_cuda(at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, bool) + 0x3e2 (0x7f0cf7f22452 in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #3: <unknown function> + 0x698f59b (0x7f0cfbb2359b in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #4: <unknown function> + 0x4356b7a (0x7f0cf94eab7a in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #5: <unknown function> + 0x611b262 (0x7f0cfb2af262 in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #6: <unknown function> + 0x4356b7a (0x7f0cf94eab7a in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #7: <unknown function> + 0x3f9750c (0x7f0cf912b50c in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #8: at::native::max_pool2d(at::Tensor const&, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, c10::ArrayRef<long>, bool) + 0xc7 (0x7f0cf912f997 in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #9: <unknown function> + 0x444d4eb (0x7f0cf95e14eb in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #10: <unknown function> + 0x61ecb5c (0x7f0cfb380b5c in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #11: <unknown function> + 0x441222a (0x7f0cf95a622a in /home/csverma/Disk/Software/MLearn/Pytorch/libtorch/lib/libtorch.so)
frame #12: <unknown function> + 0x2e671 (0x563c8699d671 in ./mnist)
frame #13: Net::forward(at::Tensor) + 0xf3 (0x563c869a21b3 in ./mnist)
frame #14: <unknown function> + 0x1eb3e (0x563c8698db3e in ./mnist)
frame #15: main + 0x707 (0x563c869895a7 in ./mnist)
frame #16: __libc_start_main + 0xe7 (0x7f0cf3b3cb97 in /lib/x86_64-linux-gnu/libc.so.6)
frame #17: _start + 0x2a (0x563c8698ae2a in ./mnist)

Aborted (core dumped)

What could be issue?

## To Reproduce

1.
1.
1.

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by izdeby** [comment]
Please, use our [forum](https://discuss.pytorch.org/) for questions. thanks
